---
layout: page
authors: ["Greg Wilson"]
title: Next-Generation Sequencing Course 2013
date: 2013-02-01
time: "10:00:00"
tags: ["Noticed", "Software Carpentry"]
---

<p><b>This post originally appeared on the <a href="https://software-carpentry.org/">Software Carpentry website.</a></b></p>
<p><strong>Analyzing Next-Generation Sequencing Data</strong></p>
<p><a href="http://bioinformatics.msu.edu/ngs-summer-course-2013">http://bioinformatics.msu.edu/ngs-summer-course-2013</a></p>
<p>June 10th&ndash;20th, 2013<br />
Kellogg Biological Station, MSU<br />
Course sponsor: NIH.</p>
<p>Instructors: Dr. C. Titus Brown, Dr. Ian Dworkin, and Dr. Istvan Albert.</p>
<p><strong>Course Description</strong></p>
<p>This intensive two week summer course will introduce attendees with a strong biology background to the practice of analyzing short-read sequencing data from Illumina and other next-gen platforms. The first week will introduce students to computational thinking and large-scale data analysis on UNIX platforms. The second week will focus on mapping, assembly, and analysis of short-read data for resequencing, ChIP-seq, and RNAseq.</p>
<p>No prior programming experience is required, although familiarity with some programming concepts is helpful, and bravery in the face of the unknown is necessary. 2 years or more of graduate school in a biological science is strongly suggested. Faculty, postdocs, and research staff are more than welcome!</p>
<p>Students will gain practical experience in:</p>
<ul>
  <li>Python and bash shell scripting</li>
  <li>cloud computing/Amazon EC2</li>
  <li>basic software installation on UNIX</li>
  <li>installing and running maq, bowtie, and velvet</li>
  <li>querying mappings and evaluating assemblies</li>
</ul>
