---
layout: page
date: 2011-03-16
time: "10:00:00"
authors: ["Greg Wilson"]
title: Next-Generation Sequencing Course at MSU
tags: ["Michigan State University", "Noticed", "Software Carpentry"]
---

<p><b>This post originally appeared on the <a href="https://software-carpentry.org/">Software Carpentry website.</a></b></p>
<p><strong>Analyzing Next-Generation Sequencing Data</strong></p>
<p>June 6th &mdash; June 17th, 2011</p>
<p>Kellogg Biological Station, Michigan State University</p>
<p>Instructors: Dr. C. Titus Brown, Dr. Ian Dworkin, and Dr. Istvan Albert.</p>
<p>Applications must be received by March 25th for full consideration.  See the full announcement at <a href="http://bioinformatics.msu.edu/ngs-summer-course-2011">http://bioinformatics.msu.edu/ngs-summer-course-2011</a> for more information.</p>
<p><strong>Course Description:</strong></p>
<p>This intensive two week summer course will introduce students with a strong biology background to the practice of analyzing short-read sequencing data from Roche 454, Illumina GA2, ABI SOLiD, Pacific Biosciences, and other next-gen platforms. The first week will introduce students to computational thinking and large-scale data analysis on UNIX platforms. The second week will focus on mapping, assembly, and analysis of short-read data for resequencing, ChIP-seq, and RNAseq.</p>
<p>No prior programming experience is required, although familiarity with some programming concepts is helpful, and bravery in the face of the unknown is necessary. 2 years or more of graduate school in a biological science is strongly suggested.</p>
<p>Students will gain practical experience in:</p>
<ul>
<li>Python and bash shell scripting</li>
<li>cloud computing/Amazon EC2</li>
<li>basic software installation on UNIX</li>
<li>installing and running maq, bowtie, and velvet</li>
<li>querying mappings and evaluating assemblies</li>
</ul>
<p>Materials from last year's course are available at <a href="http://ged.msu.edu/angus/">http://ged.msu.edu/angus/</a> under a Creative Commons/use+reuse license.</p>
